code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -143,6 +146,7 @@
 	drink_icon = "ginvodkaglass"
 	drink_name = "Glass of vodka"
 	drink_desc = "The glass contain wodka. Xynta."
+	taste_message = "comradery"
```

Comment:
camaraderie? But this misspelling makes more sense for the drink this is.
Komraderie :laughing:

Reply:
Sorry! English isn't my mother language, but I think I might add Komraderie, it fits perfectly.